tests/scala/toolchain/Parent.scala

```diff
@@ -0,0 +1,5 @@
+object Parent {
```

Comment:
Would it be worth using Scala 3 specific features in this file, so it breaks if somehow you end up with a Scala 2 toolchain here instead of 3?

Reply:
Sure, I can do that.